interface/billing/sl_eob_search.php

```diff
@@ -1071,7 +1147,19 @@ function npopup(pid) {
 
 if (!$INTEGRATED_AR) SLClose();
 ?>
-
+<tr bgcolor='<?php echo $bgcolor ?>'>
+<td>Totals</td>
```

Comment:
Hi Sherwin,
But I'm not asking to translate the variables, just the 'Totals' string.
-brady